index.html

```diff
@@ -1,94 +1,129 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="styles.css" />
-    <title>My Portfolio</title>
-  </head>
-  <body>
-    <main class="body">
-      <header id="#portfolio"  class="top">
-        <nav class="navigation_bar">
-          <a href="#">My Logo</a>
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="24"
-            height="24"
-            viewBox="0 0 24 24"
-            class="box"
-          >
-            <path
-              d="M20 11H4c-.6 0-1 .4-1 1s.4 1 1 1h16c.6 0 1-.4 1-1s-.4-1-1-1zM4 8h16c.6 0 1-.4 1-1s-.4-1-1-1H4c-.6 0-1 .4-1 1s.4 1 1 1zm16 8H4c-.6 0-1 .4-1 1s.4 1 1 1h16c.6 0 1-.4 1-1s-.4-1-1-1z"
-            />
-          </svg>
-          
-            <ul>
-              <li>Portfolio</li>
-              <li>About</li>
-              <li>Contact</li>
-              <li><img src="./img/Icon - Mail.png" alt=""></li>
-            </ul>
-          
-        </nav>
-        
-      </header>
-      <section class="menuBar">
-        <div class="closeMenu">X</div>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="styles.css" />
+  <title>My Portfolio</title>
+</head>
+
+<body>
+
+
+  <main class="body">
+    <!-- pop up mobile -->
+    <div class="popupMobile">
+      <div class="popUpData">
+        <header>
+          <p class="popH1">Multi Post Stories</p>
+          <span class="popX">&times;</span>
+        </header>
+        <section>
+          <ul class="popList">
+            <li>HTML</li>
+            <li>Bootsrap</li>
+            <li>Ruby on Rails</li>
+          </ul>
+          <div class="popImage">
+            <img src="img/popup.png" alt="10 years">
+          </div>
+          <div class="popPara">
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+              industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+              scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into
+              electronic typesetting, remaining essent</p>
+          </div>
+
+          <div class="popLinks"><button><a href="">See live</a><span><img src="img/seeLive.png" alt=""></span></button>
```

Comment:
- [ ] Great effort colleague, but as stated in the project requirement, the pop should also be created dynamically using JavaScript as you have done for the work cards, And after if has been created dynamically all the hardcoded line of codes should be remove from the HTML file.